src/components/Header/index.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+import './header.css';
+
+export default function Title(props) {
```

Comment:
You've used 4 spaces of indentation here, but 2 spaces in stories/index.js - 

Projects should be consistent - in general we stick to 2 spaces though